.github/workflows/auto_release.yaml

```diff
@@ -0,0 +1,94 @@
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          tags: true
+      - name: Install gcc-mingw-w64-x86-64
+        run: |
+             sudo apt -q update
+             sudo apt -q install --no-install-recommends gcc-mingw-w64-x86-64
+      - name: Build truckersmp-cli.exe
+        run: make
+      - name: Strip truckersmp-cli.exe
+        run: x86_64-w64-mingw32-strip truckersmp-cli.exe
+      - name: Compress files
+        id: compress_files
+        run: |
+             tag="${{ steps.set_release_tag.outputs.tag }}"
+             tardir="$(basename ${{ github.repository }})-$tag"
+             mkdir -p .output/$tardir
+             mv * .output/$tardir
+             tar cvf output.tar -C .output $tardir
+             zstd --ultra -20 output.tar -o output.tar.zst
+             xz output.tar
+             echo "::set-output name=tarfilename::${tardir}.tar"
+             echo "::set-output name=version::$tag"
+      - name: Generate changelog
+        id: generate_changelog
+        run: |
+             import subprocess as subproc
+             args = ("git", "tag", "-l", "--sort=-v:refname")
+             basetag = subproc.check_output(args).decode("utf-8").splitlines()[1]
+             args = (
+               "git", "log",
+               "--format= * [%h] %s", "--no-merges", "--reverse", basetag + "..",
+             )
+             output = subproc.check_output(args).decode("utf-8")
+             body = ""
+             if output:
+                 body += "## Changes since {}%0A".format(basetag)
+                 body += output.replace("\n", "%0A")
+             print("::set-output name=body::" + body)
+        shell: python
+      - name: Create GitHub release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.set_release_tag.outputs.tag }}
+          release_name: Release ${{ steps.compress_files.outputs.version }}
+          body: ${{ steps.generate_changelog.outputs.body }}
+          draft: false
+          prerelease: false
+      - name: Upload release asset (tar.xz)
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: output.tar.xz
+          asset_name: ${{ steps.compress_files.outputs.tarfilename }}.xz
+          asset_content_type: application/x-tar
+      - name: Upload release asset (tar.zst)
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: output.tar.zst
+          asset_name: ${{ steps.compress_files.outputs.tarfilename }}.zst
+          asset_content_type: application/x-tar
```

Comment:
I don't think we need two compressed archives here, they have the same content.
Keep xz here since you've found that's more widespread.

Reply:
Many open-source projects provide their source archive in multiple formats.
"XZ Utils" is provided in 4 formats (.tar.gz, .tar.bz2, .tar.zst, and .tar.xz).

I think we don't *need both* but don't think we should remove .tar.zst.

Reply:
Ok, can't hurt to provide both.